package/docker/Dockerfile

```diff
@@ -11,3 +13,17 @@ COPY package/docker/style.css /usr/local/apache2/htdocs/
 
 # Copy BahmniApps
 COPY ui/dist/. /usr/local/apache2/htdocs/bahmni/
+
+# Download Person Management App
+RUN <<EOR
```

Comment:
Any particular reason of packaging the person management app like this? We could have just bundled it as another image and/or used from MFE way or even proxy to the container serving the app.  
This will also mean that for every build of bahmniapps, we are downloading and packaging person management app.

Reply:
+1 here, also this would make versioning difficult, every build/patch fix in person management app would need an commit in bahmniapps also and will need a release of bahmni-web image as well.